packages/xdg_directories/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.2.0+3
+
+* Return null instead of throwing exception from xdg_directories/getUserDirectory when xdg-user-dir executable is missing.
```

Comment:
Nit: Returns

(See the repo channel style guide discussion of been form.)

Reply:
Fixed!